app/src/main/java/com/duckduckgo/app/browser/downloader/FilenameExtractor.kt

```diff
@@ -74,10 +70,7 @@ class FilenameExtractor @Inject constructor(
         mimeType: String?
     ): String {
         val tidiedUrl = url.removeSuffix("/")
-        var guessedFilename = when (appBuildConfig.flavor) {
-            BuildFlavor.INTERNAL -> DownloaderUtil.guessFileName(tidiedUrl, contentDisposition, mimeType)
-            else -> URLUtil.guessFileName(tidiedUrl, contentDisposition, mimeType)
-        }
+        var guessedFilename = DownloaderUtil.guessFileName(tidiedUrl, contentDisposition, mimeType)
```

Comment:
This used to be available only for internal builds, now it's available on all builds.